src/App.tsx

```diff
@@ -1,61 +1,127 @@
 import './App.scss';
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './components/Types/User';
+import { Todo } from './components/Types/Todo';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [todosTitle, setTodoTitle] = useState('');
+  const [todosUserId, setTodosUserId] = useState(-1);
+  const [titleIsVerified, setIsTitleVerified] = useState(true);
+  const [userIsVerified, setIsUserVerified] = useState(true);
+  const maxIdInitial = Math.max(...todosFromServer.map(todo => todo.id));
+  const [maxId, setMaxId] = useState(maxIdInitial + 1);
```

Comment:
better keep all states kinda separated from regular variables
```suggestion
  const [todosTitle, setTodoTitle] = useState('');
  const [todosUserId, setTodosUserId] = useState(-1);
  const [titleIsVerified, setIsTitleVerified] = useState(true);
  const [userIsVerified, setIsUserVerified] = useState(true);
  const [maxId, setMaxId] = useState(maxIdInitial + 1);

  const maxIdInitial = Math.max(...todosFromServer.map(todo => todo.id));

```